components/Tabs/src/MDCTabBar.h

```diff
@@ -133,9 +140,9 @@ IB_DESIGNABLE
 
 /**
  Delegate protocol for MDCTabBar. Clients may implement this protocol to receive notifications of
- selection changes in the tab bar.
+ selection changes in the tab bar or to change the bar's position-based appearance.
```

Comment:
Consider:
or return the bar's position.
(There is only one method in UIBarPositioningDelegate)